gprofiler/profilers/java.py

```diff
@@ -256,6 +256,12 @@ def read_output(self) -> Optional[str]:
             " The added buildid+offset can be resolved & symbolicated in the Performance Studio."
             " This is useful if debug symbols are unavailable for the relevant DSOs (libjvm, libc, ...).",
         ),
+        ProfilerArgument(
+            "--java-no-version-check",
+            dest="java_no_version_check",
+            action="store_true",
+            help="Skip the JDK version check (that is done before invoking async-profiler)",
+        ),
```

Comment:
We usually do `store_false` for these arguments (so the dest would be `java_version_check` and the default would be `True`) so the condition later on would be `if java_version_check` which is a bit more readable imho

Reply:
Fair